flaml/data.py

```diff
@@ -156,11 +156,11 @@ def get_output_from_log(filename, time_budget):
                 best_error_list.append(best_val_loss)
                 logged_metric_list.append(train_loss)
                 error_list.append(val_loss)
-                config_list.append({"Current Learner": learner,
-                                    "Current Sample": sample_size,
+                config_list.append({"Current Learner":          learner,
```

Comment:
flask complains these extra spaces. You can find them in "flint with flake8" section of the checks:　https://github.com/microsoft/FLAML/pull/89/checks?check_run_id=2599396437